schema/trivia.schema.json

```
@@ -0,0 +1,52 @@
+{
+  "$id": "https://raw.githubusercontent.com/Cog-Creators/Red-DiscordBot/V3/develop/schema/trivia.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Red-DiscordBot Trivia List file",
+  "type": "object",
+  "properties": {
+    "AUTHOR": {
+      "type": "string",
+      "description": "Author of the Trivia quiz."
+    },
+    "CONFIG": {
+      "type": "object",
+      "description": "The configuration for a trivia session.",
+      "properties": {
+        "bot_plays": {
+          "type": "boolean",
+          "description": "Whether or not the bot gains points during the session. Defaults to true."
+        },
+        "delay": {
+          "type": ["integer", "number"],
+          "description": "The maximum number of seconds permitted to answer a question, must be a positive number greater than or equal to 4.0. Defaults to 15.0.",
+          "minimum": 4.0
+        },
+        "max_score": {
+          "type": "integer",
+          "description": "Number of points required in order to win the trivia, must be a positive integer. Defaults to 10.",
+          "exclusiveMinimum": 0
+        },
+        "payout_multiplier": {
+          "type": ["integer", "number"],
+          "description": "The payout multiplier, must be a positive number or zero. Defaults to 0.0.",
+          "minimum": 0
+        },
+        "reveal_answer": {
+          "type": "boolean",
+          "description": "Whether or not to reveal the answer when the question times out. Defaults to true.",
+          "default": true
+        },
+        "timeout": {
+          "type": ["integer", "number"],
+          "description": "Number of seconds that need to pass until trivia stops due to no response, must be a positive number greater than 0.0. Defaults to 120.0.",
+          "exclusiveMinimum": 0
+        },
+        "use_spoilers": {
+          "type": "boolean",
+          "description": "Whether to hide the answers in spoilers when revealing the question's answers. Defaults to false."
+        }
+      },
+      "additionalProperties": false
+    }
+  }
```

This remains to be unaddressed:
> - schema should specify that all other keys should be lists of strings
>     - this should be achievable with `additionalProperties`: https://json-schema.org/draft/2020-12/json-schema-core.html#section-10.3.2.3

This should work:
```suggestion
  },
  "additionalProperties": {
    "type": "array",
    "items": {
      "type": "string"
    }
  }
```